04-gas/gas-section-five/gas-section-five.css

```diff
@@ -1,59 +1,147 @@
-  
-.gas-section-five-subcontainer {
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-template-areas: 
-    "gas-section-five-copy-container"
-    "gas-section-five-img-container";
-    justify-content: center;
-    row-gap: var(--items-gap-mobile);
-    padding: var(--subcontainer-padding-mobile);
+
+.gas-section-five-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
+  background-color: var(--level1-background-color);
 }
-  
-.gas-section-five-subcontainer > * {
-    border: var(--subcontainer-border-mobile);
+
+@media (min-width:600px) {
+  .gas-section-five-container {
+    padding: var(--container-padding-desktop);
+  }
 }
 
-@media (min-width:600px){
- .gas-section-five-subcontainer{
-    display: grid;
-    grid-template-columns: repeat(2,1fr);
-    grid-template-areas: 
-    "gas-section-five-img-container gas-section-five-copy-container";
-    }
+.gas-section-five-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+   "gas-section-five-copy-container"
+   "gas-section-five-image-container";
+  row-gap: var(--subcontainer-row-gap-mobile);
+  column-gap: none;
+  padding: var(--subcontainer-padding-mobile);
+  border: var(--subcontainer-border-mobile);
+  background-color: var(--level2-background-color);
+}
 
+@media (min-width:600px) {
+  .gas-section-five-subcontainer{
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+    "gas-section-five-copy-container gas-section-five-image-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+    padding: var(--subcontainer-padding-desktop);
+  }
 }
 
 .gas-section-five-copy-container {
-    grid-area: gas-section-five-copy-container;
-    display: grid;
-    justify-content: center;
+  grid-area: gas-section-five-copy-container;
+  display: grid;
+  align-content: start;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 600px) {
+  .gas-section-five-copy-container {
     align-content: center;
-    text-align: center;
-    row-gap: var(--items-gap-mobile);
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
 }
+
+.gas-section-five-title-container{
   
+}
+
+@media (min-width: 600px){
+  .gas-section-five-title-container{
+  }
+}
+
 .gas-section-five-title {
-    font-size: var(--title-size-mobile);
-    background-color: var(--title-background-color-mobile);
+  display: flex;
+  flex-direction: column;
+  row-gap: var(--title-container-row-gap-mobile);
+  text-align: center;
+  font-size: var(--gas-title-font-size-mobile);
+  font-weight: var(--gas-title-font-weight-mobile);
+}
+
+@media (min-width: 600px) {
+  .gas-section-five-title {
+    flex-direction: row;
+    column-gap: var(--title-container-column-gap-desktop);
+    row-gap: none;
+    text-align: left;
+    font-size: var(--gas-title-font-size-desktop);
+    font-weight: var(--gas-title-font-weight-desktop);
+  }
```

Comment:
i assume 
--title-container-row-gap-mobile
--title-container-column-gap-desktop
--gas-title-font- 

etc..

is in your updated global-styles.
i can't see how this looks when applied to check it.

if the title-container rule is specific to gas then maybe you can prefix it with gas

it depends if you have another use-case elsewhere for it

Reply:
here they must be different with other page and hero section. for now we can keep like this